src/plugins/elasticsearch/lib/filter_headers.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+
+module.exports = function (originalHeaders, headersToKeep) {
```

Comment:
Use `export default` here instead of `module.exports =`.  We're moving away from the commonjs module syntax in favor of the native module syntax.